### DISCUSSION TITLE
Add Mastodon as a new commenting system

### DISCUSSION THREAD

**Discussion by KHannibal** [Lednerb/bilberry-hugo-theme#467] [Ideas]
Hi, and happy new year ! 🥳 

I've read on multiple websites that Mastodon can be used as a commenting system.

- https://f1rum.fr/posts/hugo-masto-comments
- https://amnesiak.org/post/2021/01/30/hugo-blog-with-mastodon-comments-extended
- https://carlschwan.eu/2020/12/29/adding-comments-to-your-static-blog-with-mastodon

It could be interesting to provide this feature by default, what do you think ?

I'll try on my blog when I've time.

Thanks !

**Comment by igor-baiborodine** [comment]
@KHannibal, it looks interesting, but the commenting engine provided by Mastodon should satisfy the [following criteria](https://github.com/Lednerb/bilberry-hugo-theme#comments) to be integrated with the Bilberry Hugo theme: 

![image](https://user-images.githubusercontent.com/1027701/210564283-1ce354cf-5390-4a9f-954b-87d1a26f8e5c.png)

**Reply by KHannibal** [reply]
Too bad. 🫠
